tools/src/test/scala/scala/scalanative/FrameworkTest.scala

```diff
@@ -33,4 +33,19 @@ class FrameworkTest extends CodeGenSpec with Matchers {
         defNames should contain(Global.Top("B$"))
     }
   }
+
+  "The test framework" should "be able to compile double quotes in names" in {
+    link(
+      "A$",
+      """object A {
+        |  val `"` = 42
+        |  def main(args: Array[String]): Unit =
+        |    println("Hello, world! " + `"`)
+        |}""".stripMargin
+    ) {
+      case (_, res) =>
+        val defNames = res.defns map (_.name)
+        defNames should contain(Global.Top("A$"))
+    }
+  }
```

Comment:
Link tests don't run LLVM part of the toolchain. For this test it's better to just put that code into `IssuesSuite.scala`.